src/test/scala/io/dockstore/Requests.scala

```diff
@@ -103,6 +103,16 @@ object Requests {
       if (token != null && !token.isEmpty) builder.headers(authHeader(token)) else builder
     }
 
+    /**
+     * Requires session attributes encodedTrsId, version, and descriptorType be set
+     * @return
+     */
+
+    def getPlainDescriptor() = {
+      val path = "/api/api/ga4gh/v2/tools/#{encodedTrsId}/versions/#{version}/PLAIN_#{descriptorType}/descriptor"
```

Comment:
don't know if it matters, but this is testing trs v2 beta, not trs v2 final

Reply:
Grr, that wasn't the intent.

Reply:
FWIW, Terra is still v2 beta I thought, so maybe more realistic. 
v2 final is what we care about for the others like DNAstack, could also load test a mix of the two
either way is probably ok in the broader strokes

Reply:
I'm pretty sure Terra is using v1.

We should move the other tests to v2 final, but I wanted to limit the scope of this PR for the new test.